addons/website/views/res_config_settings_views.xml

```diff
@@ -80,7 +80,7 @@
                         <setting id="website_cookies_bar_setting" help="Display a customizable cookies bar on your website">
                             <field name="website_cookies_bar"/>
                         </setting>
-                        <setting id="shared_user_account_setting" groups="website.group_multi_website" help="Accounts are usable across all your multiple websites">
```

Comment:
It's probably not what you want as this will almost never be shown since this is only shown when you have 2 website, and it's linked to the wrong setting anyway ^^
You probably meant to add it on line 80 and not 83?